src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -3,5 +3,6 @@
     public class OpenTelemetryCollectorSinkConfiguration
     {
         public string CollectorUri { get; set; }
+        public string CollectorProtocol { get; set; }
```

Comment:
I think this approach is better, but let's do this:
```suggestion
        public CollectorInfo Collector { get; set; }
```

Reply:
Added